src/host/sys/macos/uptime.rs

```diff
@@ -0,0 +1,22 @@
+use nix::libc::{c_void, sysctl, timeval};
```

Comment:
Move this line below the std imports and include an empty line separating them too.